prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml

```diff
@@ -62,6 +62,33 @@ postsubmits:
         - boskos
         - deploy
 
+  - name: post-test-infra-push-prowbazel
+    cluster: test-infra-trusted
+    run_if_changed: '^docker/prowbazel/Makefile$'
+    decorate: true
+    max_concurrency: 1
+    branches:
+    - master
+    labels:
+      preset-prow-pusher-service-account: "true"
+      preset-dind-enabled: "true"
+    annotations:
+      testgrid-create-test-group: "false"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191017-ac4b4b5-master
```

Comment:
This seems like the wrong image. Why can't we use a standard istio image here?

Reply:
This is the image I am familiar with for working with docker-in-docker. I will ask @howardjohn if there is an Istio image for this.

Reply:
Updated to using the `gcr.io/istio-testing/build-tools` image.